recipe/meta.yaml

```
@@ -20,7 +20,7 @@ requirements:
     - pip
   run:
     - python >=3.10,<3.13
-    - openmm >=7.6.0
+    - openmm >=8.1.2
```

Some preference for keeping this a little tighter
```suggestion
    - openmm ~=8.1.2
```